package.json

```diff
@@ -147,8 +148,8 @@
     "predeploy": "yarn run build-gh && cp styleguide/index.html styleguide/404.html && cp favicon.png styleguide/favicon.png",
```

Comment:
Note that above "postinstall" still calls `npm rebuild node-sass`. I'm not sure what component is still using `node-sass`. `yarn why` tells us that only the `rollup-plugin-scss` is using it (which is overridden in the rollup config), but I ran into issues when running `yarn start`.